test-tools/wamr-ide/VSCode-Extension/formatters/rust.py

```diff
@@ -0,0 +1,746 @@
+'''
+Copyright (c) 2016 Vadim Chugunov
```

Comment:
Should also add another line:
`SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception`

Reply:
done